athlinks-schema.ttl

```diff
@@ -147,8 +162,47 @@ athlinks:pace
   a owl:DataTypeProperty ;
   rdfs:range schema:DateTime .
 
+# For  AthleticRace PROPERTIES
+athlinks:giveaway
+  a owl:DataTypeProperty;
+  rdfs:range xsd:string .
+
+athlinks:require_phone
+  a owl:DataTypeProperty;
+  rdfs:range xsd:boolean .
+
+athlinks:require_dob
+  a owl:DataTypeProperty;
+  rdfs:range xsd:boolean .
```

Comment:
these requre* fields should be part of some kind of registration requirements object. Not as part of the Race object.